### PR TITLE
Create Utility function to check if folder/file exists or not, if not create that folder/file

### PR DESCRIPTION
**Pull request by 100rabhpawar**
Story details: https://app.shortcut.com/rebase-it-labs/story/481

### CODE REVIEW

File: src/utils/fileCheck.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  if (!fs.existsSync('./database')) {
+    fs.mkdirSync('database');
+  }
+
+  if (!fs.existsSync(path)) {
+    fs.createWriteStream(path, { flags: 'wx' });
+  }
+
+  if (data) {
+    fs.writeFileSync(path, JSON.stringify(data));
```
Comment (prasanarebase) [inline src/utils/fileCheck.ts:14]: will the data always be JSON?

File: src/utils/fileCheck.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// TO check if file/folder exists or not, if not create file/folder.
+import * as fs from 'fs';
+
+function fileCheck(path: string, data?: any) {
+  if (!fs.existsSync('./database')) {
+    fs.mkdirSync('database');
+  }
+
+  if (!fs.existsSync(path)) {
```
Comment (prasanarebase) [inline src/utils/fileCheck.ts:9]: Add comments what this is doing

File: src/utils/fileCheck.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// TO check if file/folder exists or not, if not create file/folder.
```
Comment (prasanarebase) [inline src/utils/fileCheck.ts:1]: Add test case

**Review by prasanarebase — CHANGES_REQUESTED** [review]

File: src/utils/fileCheck.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import * as fs from 'fs';
+
+function fileCheck(path: string, data?: any) {
+  if (!fs.existsSync('./database')) {
```
Comment (prasanarebase) [inline src/utils/fileCheck.ts:5]: why is this checking database folder with static value

**Review by prasanarebase — CHANGES_REQUESTED** [review]

**Review by praveenrebase — APPROVED** [review]

**Pull request merged**